qiita_db/test/test_meta_util.py

```diff
@@ -299,6 +299,8 @@ def _get_daily_stats():
 
         for k, exp, f in vals:
             redis_key = '%s:stats:%s' % (portal, k)
+            print("here's redis", f(redis_key))
```

Comment:
can you remove these prints ?

Reply:
I've been guilty of doing this myself. One alternative that can work better is to use nosetests' variant of pdb; by adding the following line: from nose.tools import set_trace; set_trace(), you can begin watching variables, stepping through functionality, etc.